### ISSUE TITLE
Need <pack>false</pack>

### ISSUE THREAD

**Issue by ericsink** [ericsink/SQLitePCL.raw#140]
See aspnet/Microsoft.Data.SQLite#320, especially comments therein from @natemcmaster

Worry about Nuget/home#4432 and dotnet/sdk#847

**Comment by natemcmaster** [comment]
See also https://github.com/dotnet/sdk/issues/843

**Comment by natemcmaster** [comment]
I can confirm that at least one problem here is that when creating a .NET Framework class library that depends on e_sqlite3.v110_xp, "dotnet pack" will end up putting runtimes\win7-x86\native\e_sqlite3.dll into the nuget package under "content" and "contentFiles".


Repro:
```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net451</TargetFramework>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="sqlitepclraw.bundle_green" Version="1.1.2"/>
  </ItemGroup>

</Project>
```
dotnet restore
dotnet pack

(Using [.NET Core CLI 1.0.0-rc4-004771](https://github.com/dotnet/core/blob/master/release-notes/rc4-download.md)).

**Comment by ericsink** [comment]
@natemcmaster You mentioned (over in the other repo PR) that the fix for this might be as simple as adding `<pack>false</pack>` to the content items.  I would like to understand how this tag works.  Can you point to any docs or information?

**Comment by bricelam** [comment]
See [NuGet pack and restore as MSBuild targets](https://docs.microsoft.com/en-us/nuget/schema/msbuild-targets)

**Comment by ericsink** [comment]
I've confirmed that the the addition of `<pack>false</pack>` does fix the specific problem mentioned above (native dlls ending up in dotnet pack).

**Comment by ericsink** [comment]
Changing the title of this issue to make it more specific.  Any other VS 2017 problems deserve their own issues.

**Comment by ericsink** [comment]
Fixed in 1.1.3
